scapy/dadict.py

```diff
@@ -11,13 +11,14 @@
 from __future__ import print_function
 from scapy.error import Scapy_Exception
 import scapy.modules.six as six
+from scapy.compat import *
 
 ###############################
 ## Direct Access dictionary  ##
 ###############################
 
 def fixname(x):
-    if x and x[0] in "0123456789":
+    if x and str(x[0]) in "0123456789":
```

Comment:
Below there's a use of `plain_str` to ensure every thing is stored as str. I think it's easier to load strings from files.
Can be discussed :/